packages/data-explorer/src/types/ApplicationState.ts

```diff
@@ -3,11 +3,65 @@ export type Toast = {
   message: string
 }
 
+type DataApiResponseItem = {
```

Comment:
suggest to split up server communication types ( maybe even share across apps) from true frontend app types

Reply:
cart will get its own view in an other pr